www-servers/caddy/caddy-2.2.0.ebuild

```diff
@@ -0,0 +1,991 @@
+	"google.golang.org/protobuf v1.23.0/go.mod"
+	"google.golang.org/protobuf v1.23.1-0.20200526195155-81db48ad09cc/go.mod"
+	"google.golang.org/protobuf v1.24.0"
+	"google.golang.org/protobuf v1.24.0/go.mod"
+	"gopkg.in/airbrake/gobrake.v2 v2.0.9/go.mod"
+	"gopkg.in/alecthomas/kingpin.v2 v2.2.6/go.mod"
+	"gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15"
+	"gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15/go.mod"
+	"gopkg.in/cheggaaa/pb.v1 v1.0.28/go.mod"
+	"gopkg.in/errgo.v2 v2.1.0/go.mod"
+	"gopkg.in/fsnotify.v1 v1.4.7/go.mod"
+	"gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2/go.mod"
+	"gopkg.in/inf.v0 v0.9.1/go.mod"
+	"gopkg.in/ini.v1 v1.51.0/go.mod"
+	"gopkg.in/ini.v1 v1.51.1/go.mod"
+	"gopkg.in/natefinch/lumberjack.v2 v2.0.0"
+	"gopkg.in/natefinch/lumberjack.v2 v2.0.0/go.mod"
+	"gopkg.in/resty.v1 v1.12.0/go.mod"
+	"gopkg.in/square/go-jose.v2 v2.4.0"
+	"gopkg.in/square/go-jose.v2 v2.4.0/go.mod"
+	"gopkg.in/square/go-jose.v2 v2.5.1"
+	"gopkg.in/square/go-jose.v2 v2.5.1/go.mod"
+	"gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7"
+	"gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7/go.mod"
+	"gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7/go.mod"
+	"gopkg.in/yaml.v2 v2.2.1/go.mod"
+	"gopkg.in/yaml.v2 v2.2.2/go.mod"
+	"gopkg.in/yaml.v2 v2.2.3/go.mod"
+	"gopkg.in/yaml.v2 v2.2.4/go.mod"
+	"gopkg.in/yaml.v2 v2.2.5/go.mod"
+	"gopkg.in/yaml.v2 v2.2.7/go.mod"
+	"gopkg.in/yaml.v2 v2.3.0"
+	"gopkg.in/yaml.v2 v2.3.0/go.mod"
+	"grpc.go4.org v0.0.0-20170609214715-11d0a25b4919/go.mod"
+	"honnef.co/go/tools v0.0.0-20180728063816-88497007e858/go.mod"
+	"honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod"
+	"honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod"
+	"honnef.co/go/tools v0.0.0-20190418001031-e561f6794a2a/go.mod"
+	"honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc/go.mod"
+	"honnef.co/go/tools v0.0.1-2019.2.3"
+	"honnef.co/go/tools v0.0.1-2019.2.3/go.mod"
+	"howett.net/plist v0.0.0-20181124034731-591f970eefbb"
+	"howett.net/plist v0.0.0-20181124034731-591f970eefbb/go.mod"
+	"mvdan.cc/interfacer v0.0.0-20180901003855-c20040233aed/go.mod"
+	"mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b/go.mod"
+	"mvdan.cc/unparam v0.0.0-20190209190245-fbb59629db34/go.mod"
+	"mvdan.cc/unparam v0.0.0-20190720180237-d51796306d8f/go.mod"
+	"mvdan.cc/unparam v0.0.0-20191111180625-960b1ec0f2c2/go.mod"
+	"rsc.io/binaryregexp v0.2.0/go.mod"
+	"rsc.io/quote/v3 v3.1.0/go.mod"
+	"rsc.io/sampler v1.3.0/go.mod"
+	"sigs.k8s.io/yaml v1.1.0/go.mod"
+	"sourcegraph.com/sourcegraph/go-diff v0.5.0/go.mod"
+	"sourcegraph.com/sqs/pbtypes v0.0.0-20180604144634-d3ebe8f20ae4/go.mod"
+	"sourcegraph.com/sqs/pbtypes v1.0.0/go.mod"
+	)
+go-module_set_globals
+SRC_URI="https://github.com/caddyserver/caddy/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	${EGO_SUM_SRC_URI}"
+
+LICENSE="Apache-2.0"
```

Comment:
This looks good, but you need to include licenses for all included libs in this ebuild. 
https://packages.gentoo.org/packages/search?q=go-license-detector there are bunch of tools to help you.

Reply:
Oh damn. I thought go ego vendor does that like cargo ebuild.
will do this soon. Thanks.

Reply:
done